src/main/java/org/isf/priceslist/manager/PriceListManager.java

```diff
@@ -42,97 +42,100 @@
 
 	@Autowired
 	private PricesListIoOperations ioOperations;
-	
+
 	/**
-	 * Return the list of {@link PriceList}s in the DB
-	 * @return the list of {@link PriceList}s
-	 * @throws OHServiceException 
+	 * Return the list of {@link List}s in the DB
+	 *
+	 * @return the list of {@link List}s
+	 * @throws OHServiceException
 	 */
 	public ArrayList<PriceList> getLists() throws OHServiceException {
-        return ioOperations.getLists();
+		return ioOperations.getLists();
 	}
-	
+
 	/**
 	 * Return the list of {@link Price}s in the DB
+	 *
 	 * @return the list of {@link Price}s
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
 	public ArrayList<Price> getPrices() throws OHServiceException {
-        return ioOperations.getPrices();
+		return ioOperations.getPrices();
 	}
 
 	/**
-	 * Updates all {@link Price}s in the specified {@link PriceList}
-	 * @param list - the {@link PriceList}
+	 * Updates all {@link Price}s in the specified {@link List}
+	 *
+	 * @param list - the {@link List}
```

Comment:
This also

Reply:
I believe I've resolved (merged) the changes from OP-399